### ISSUE TITLE
FileManager menu: split first menu into two?

### ISSUE THREAD

**Issue by poire-z** [koreader/koreader#3802]
Just a thought, that I somehow just got today...
The first (gear icon) top menu of FileManager is the only one that spans more than 1 page.
It could be splitted into 2:
1 - (with the filemanager icon we see in reader top menu), with items from `Display mode` to `Start with:`, stuff only related to the File manager.
2 - (with the gear icon) with items `Frontlight` to `Time and date`, nearly similar to the one in reader top menu.
(That would make 4+1 (instead of 3+1) top menu icons for filemanager - and reader menu 6+1)

**Comment by Frenzie** [comment]
Yup. ;-)

**Comment by poire-z** [comment]
(If you feel like doing it, please do - you're more familiar with menu than I am :)

**Comment by Frenzie** [comment]
I'm trying to avoid doing as much as I can. :-P

What icon? The same file cabinet that "exits" the reader?

**Comment by poire-z** [comment]
Yes, that one. Unless it looks confusing and would make one feel it would "exit" FileManager too.

**Comment by Frenzie** [comment]
Alright, but I don't promise I'll take a look today. ;-)

**Comment by robert00s** [comment]
https://github.com/koreader/koreader/issues/3578#issuecomment-359075100
:)

**Comment by Frenzie** [comment]
Gah, not the pokeball again! :-P

**Comment by poire-z** [comment]
Oups, didn't remember it was already suggested :)
But indeed, not that pokeball icon that express nothing.

**Comment by robert00s** [comment]
- double gear? 
https://commons.wikimedia.org/wiki/File:Gear_-_Noun_project_7137.svg

![obraz](https://user-images.githubusercontent.com/22982594/37927535-b5f618f4-313a-11e8-9a6d-218f755cd00f.png)

- gear wrench?
https://www.shareicon.net/options-setup-cogwheel-gear-wrench-708830
![obraz](https://user-images.githubusercontent.com/22982594/37927854-67bde0e4-313b-11e8-9553-aefc7b55ba11.png)

**Comment by poire-z** [comment]
Let's see how it looks with the "file cabinet" icon, which for all of us already means File manager, so that's where we'd find the File manager settings.

**Comment by Frenzie** [comment]
@robert00s Before looking elsewhere note that the icons come from a set over at https://github.com/Templarian/WindowsIcons

**Comment by cramoisi** [comment]
@poire-z @Frenzie perhaps nothing but at boot with latest, got a warning in crash.log,

WARN menu id not found : plus_menu

**Comment by Frenzie** [comment]
Latest like today's latest? Should've been fixed in #3844.

**Comment by cramoisi** [comment]
more like yesterday latest, due to timezone.

**Comment by Frenzie** [comment]
Fine, I should've said "do you mean" `v2015.11-1642-gab712f1_2018-04-08`. :-P But they're actually built early in the morning for us, around 6:00 CET. The date comes from the last included commit, so even if you built it a thousand years from now it'd still say 2018-04-08.

But `v2015.11-1640-gd7a7a78_2018-04-07` would do too, because d7a7a78 is the very commit supposed to fix it.

**Comment by cramoisi** [comment]
@Frenzie I'm on 1640, and I got the warning.

**Comment by cramoisi** [comment]
Still got it , btw : 
(looking after my reading lunch session : (and I got some error I don't know from where, the book wasn't even opened with muPDF so it should be from the menu ?)

```
**04/12/18-12:43:25 WARN  menu id not found: plus_menu**
error: end of data in attribute value
04/12/18-12:50:37 WARN  HTML loading error: ./ffi/mupdf.lua:67: MuPDF cannot open document from text: end of data in attribute value (2)
error: end of data in attribute value
04/12/18-12:50:48 WARN  HTML loading error: ./ffi/mupdf.lua:67: MuPDF cannot open document from text: end of data in attribute value (2)
```

**Comment by Frenzie** [comment]
Did you use the dictionary?

**Comment by cramoisi** [comment]
I guess I use it at least once between 12 and 13.30, yes

**Comment by Frenzie** [comment]
Then that's your mupdf error. :)

**Comment by cramoisi** [comment]
thanks :) I remember opening it from a word inside the lookup history

**Comment by poire-z** [comment]
I updated yesterday to 1646-g305e75c_2018-04-10, and it looked to me I didn't get the `menu id not found` logged anymore, after some early switches from reader to filemanager (haven't looked after that if I had them in other circumstances as it seemed fixed to me).

**Comment by cramoisi** [comment]
mmh, I updated yest morning to 1646 too and it's where these lines come from.

**Comment by poire-z** [comment]
Original issue fixed in #3836, tap_menu warning solved by #3903.
